.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests in Centos container
+
+on: [pull_request, push]
+
+jobs:
+  tests-centos-8:
+    name: "Unittests on Centos 8"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

I know we use v2 in other places/repos as well, but it's [seriously outdated](https://github.com/actions/checkout/releases/tag/v2.0.0). [Let's use v4](https://github.com/marketplace/actions/checkout)!

Reply:
True, I will fix this one.